doc/api/util.md

```diff
@@ -10,6 +10,67 @@ module developers as well. It can be accessed using:
 const util = require('util');
 ```
 
+## util.callbackify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function} An `async` function
+* Returns: {Function} a callback style function
+
+This method converts Promise returning API endpoints to ones that use callbacks.
```

Comment:
Do we even need this sentence? As @addaleax said, it's basically covered by the next sentence. I think:

```
Takes an `async` function (or a function that returns a Promise)
and returns a function following the Node.js error first callback style,
```

is a perfect introduction to this function.